scripting/pugsetup/util.sp

```diff
@@ -3,7 +3,7 @@
 
 #tryinclude "manual_version.sp"
 #if !defined PLUGIN_VERSION
-#define PLUGIN_VERSION "2.0.8-dev"
+#define PLUGIN_VERSION "2.0.9-dev"
```

Comment:
This shouldn't be changing here. 2.0.8 hadn't been releases, and there's no reason to skip it.